htdocs/product/stock/class/api_warehouses.class.php

```diff
@@ -71,10 +71,10 @@ public function get($id)
 			throw new RestException(404, 'warehouse not found');
 		}
 
-		if (!DolibarrApi::_checkAccessToResource('warehouse', $this->warehouse->id)) {
+/*		if (!DolibarrApi::_checkAccessToResource('warehouse', $this->warehouse->id)) {
```

Comment:
This disable the security on warehouse objects.

Reply:
I think it is better to disable momentarily this check as it is bugged in respect of wirehouses. There remains a first security check to check the user read right on warehouses.